core/tests/protractor_desktop/wipeout.js

```diff
@@ -46,6 +46,11 @@ describe('When account is deleted it', function() {
     await deleteAccountPage.requestAccountDeletion('userToDelete1');
     expect(await browser.getCurrentUrl()).toEqual(
       'http://localhost:9001/pending-account-deletion');
+    await users.login('user1@delete.com');
+    await browser.waitForAngularEnabled(false);
```

Comment:
Can you re-enable waiting for angular after we've left the sign-up page?

Reply:
I removed it because I realized it is not needed since the user will get redirected to the pending deletion page where Angular works correctly.